pkg/cmd/gist/view/view.go

```diff
@@ -60,6 +70,21 @@ func NewCmdView(f *cmdutil.Factory, runF func(*ViewOptions) error) *cobra.Comman
 
 func viewRun(opts *ViewOptions) error {
 	gistID := opts.Selector
+	client, err := opts.HttpClient()
+	if err != nil {
+		return err
+	}
+
+	if gistID == "" {
+		gistID, err = promptGists(client)
+		if err != nil {
+			return err
+		}
+
+		if gistID == "" {
+			return nil
```

Comment:
Since `gist list` simply return when there are no any gists, so I keep the behavior consistent in `gist` command.
Or message like "There are no gists from [username]" should be returned?

Reply:
I think a message saying that the user has no gists would be a nice improvement, here.

Reply:
Would it be better to display current username? Or just "There are no gists from current user".
If showing username is preferred, is there any way to access current username other than passing in parameter to `viewRun`?